Sources/Template.swift

```diff
@@ -14,7 +14,7 @@ open class Template: ExpressibleByStringLiteral {
   public let name: String?
 
   /// Create a template with a template string
-  public required init(templateString: String, environment: Environment? = nil, name: String? = nil) {
+  public required init(templateString: String, environment: Environment? = nil, name: String? = nil) throws {
```

Comment:
@kylef If so, why do you `try` it on Line no.55 and 68 in `Loader.swift`? It made me believe that you are doing it intentionally and i ended up making this throwable.